vnext/Desktop.IntegrationTests/WebSocketModuleIntegrationTest.cpp

```diff
@@ -1,25 +1,28 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// clang-format off
```

Comment:
Same here. We should remove the global suppressions. If macros are causing issues, you can add it to `ForEachMacros` in the .clang-format file.